### DISCUSSION TITLE
Error after running make: Linking CXX executable  ../../bin/yarpmanager

### DISCUSSION THREAD

**Discussion by pamely** [robotology/community#652] [Q&A] [answered]
After running  the `make`  command I got the following error: 

```console
[ 91%] Linking CXX executable ../../bin/yarpmanager
/usr/bin/ld: /lib/x86_64-linux-gnu/libedit.so.2: undefined reference to `tgetnum@NCURSES6_TINFO_5.0.19991023'
/usr/bin/ld: /lib/x86_64-linux-gnu/libedit.so.2: undefined reference to `tputs@NCURSES6_TINFO_5.0.19991023'
/usr/bin/ld: /lib/x86_64-linux-gnu/libedit.so.2: undefined reference to `tgoto@NCURSES6_TINFO_5.0.19991023'
/usr/bin/ld: /lib/x86_64-linux-gnu/libedit.so.2: undefined reference to `tgetent@NCURSES6_TINFO_5.0.19991023'
/usr/bin/ld: /lib/x86_64-linux-gnu/libedit.so.2: undefined reference to `tgetstr@NCURSES6_TINFO_5.0.19991023'
/usr/bin/ld: /lib/x86_64-linux-gnu/libedit.so.2: undefined reference to `tgetflag@NCURSES6_TINFO_5.0.19991023'
collect2: error: ld returned 1 exit status
make[2]: *** [src/yarpmanager/CMakeFiles/yarpmanager.dir/build.make:668: bin/yarpmanager] Error 1
make[1]: *** [CMakeFiles/Makefile2:8171: src/yarpmanager/CMakeFiles/yarpmanager.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

Please, what is reason for this? 

Pamely

**Comment by traversaro** [comment, marked as answer]
We had a similar issue in https://github.com/robotology/robotology-superbuild/issues/1168 . Do you happen to have a conda environment or another similar environment activated? In general, when building C/C++ software with dependencies installed via apt, it is important to make sure to de-activate any existing conda environment that is activated, as conda environments ship their own version of some of the C/C++ libraries installed by apt, and this can lead to incompatibilities like the one that you see. 

If indeed you have a conda environemnt activated, I suggest you to remove the existing build directory, deactivate the conda environment and re-do the build again.

If you like to use conda, an alternative for you may be to directly install the `yarp` conda package available in the `conda-forge` channel: https://github.com/conda-forge/yarp-feedstock .

**Reply by pamely** [reply]
Thank you @traversaro. As you suspected, I was building the YARP in a conda environment. I deactivated it and running and everything is fine. I prefer this option than installing the YARP conda package. Thank you. 
Pamely
